src/link.js

```diff
@@ -36,8 +36,11 @@ const linkDependencyAndroid = (androidProject, dependency) => {
     log.info(`Android module ${dependency.name} is already linked`);
     return null;
   }
+  
+  // Legacy support for top-level params that is Android-only
```

Comment:
I though the idea is that dependency.config.params would be a "common" params for both platforms. Why you call it a legacy for Android?